cmd/soroban-cli/src/fee.rs

```diff
@@ -16,15 +17,39 @@ pub struct Args {
     /// Number of instructions to simulate
     #[arg(long, help_heading = HEADING_RPC)]
     pub instructions: Option<u32>,
+    /// Build the transaction only write the base64 xdr to stdout
+    #[arg(long, help_heading = HEADING_RPC)]
+    pub build_only: bool,
+    /// Simulation the transaction only write the base64 to stdout
+    #[arg(long, help_heading = HEADING_RPC, conflicts_with = "build_only")]
+    pub sim_only: bool,
```

Comment:
Why do the new options get attached to the fee instead of to the `commands::config::Args` type which is where the --source-account is attached today? These options feel more similar to source account than fees. Fees could also arguably live in the same place as source account, so maybe this is an area that would benefit of a refactor.

Reply:
I missed the comment here https://github.com/stellar/soroban-cli/issues/1265#issuecomment-2009578081, makes sense, I'll continue that conversation over there.